### ISSUE TITLE
Two directories in BLT MPI link path cause -rpath to appear in quotes

### ISSUE THREAD

**Issue by sshudler** [LLNL/blt#363]
Hi,

I've encountered this issue on Quartz when compiling Conduit through Ascent's uberenv.
In this particular case I'm using my local compilation of MPICH. Here are the excerpts from the compilation log:
-- BLT Version: 0.3.0
-- CMake Version: 3.14.5
-- BLT MPI Link Flags:     -Wl,-rpath -Wl,/g/g92/shudler1/install/mpich/lib;-Wl,-rpath -Wl,/g/g92/shudler1/install/mpich/lib
...
[  6%] Linking CXX executable ../../../tests/blt_mpi_smoke
...
/usr/tce/packages/gcc/gcc-4.9.3/bin/g++      -O2 -g -DNDEBUG  -rdynamic -Wl,-rpath -Wl,/g/g92/shudler1/install/mpich/lib "-Wl,-rpath -Wl,/g/g92/shudler1/install/mpich/lib" CMakeFiles/blt_mpi_smoke.dir/blt_mpi_smoke.cpp.o  -o ../../../tests/blt_mpi_smoke -Wl,-rpath,/g/g92/shudler1/install/mpich/lib /g/g92/shudler1/install/mpich/lib/libmpi.so /g/g92/shudler1/install/mpich/lib/libmpicxx.so /g/g92/shudler1/install/mpich/lib/libmpifort.so

It looks like the link parameters after ';' are put inside quotes.
When I use the Mvapich2 installation this issue doesn't happen because there is only one directory in the flags:
-- BLT MPI Link Flags:     -Wl,-rpath -Wl,/usr/tce/packages/mvapich2/mvapich2-2.2-gcc-4.9.3/lib

This issue looks related to #266 

Thanks,
Sergei

**Comment by white238** [comment]
Are you on the newest BLT develop?  Some recent fixes were put in place to solve this and by looking at your link line you might be in an old state...

**Comment by sshudler** [comment]
No, it appears I'm using the master branch (right at commit 2c19277). That's the BLT version pointed by Conduit's master branch which I'm compiling from.

**Comment by white238** [comment]
Try develop, some recent fixes went in to get cmake 3.14.5 working on various projects.

**Comment by cyrush** [comment]
@white238 Conduit is using BLT 0.3.0, should we move to a newer version?

**Comment by cyrush** [comment]
@sshudler -- for ascent, which spack config setup are you using?

I suggest using:
`scripts/uberenv/spack_configs/llnl/pascal_openmp/`

To point to the proper MPI.

**Comment by white238** [comment]
develop is the only one with the specific CMake 3.14.5 fixes (its been less than a month since those were implemented).  They have been working in various projects (Axom, Serac, etc.)
